src/docs/components/events.md

```diff
@@ -89,7 +89,7 @@ export class TodoList {
 
 ## Listen Decorator
 
-The `Listen()` decorator is for listening DOM events, including the ones dispatched from `@Events`.
+The `Listen()` decorator is for listening for DOM events, including the ones dispatched from `@Events`.
```

Comment:
Thanks for submitting this correction! I'm totally nitpicking here, but do you think "to DOM events" sounds better?

Reply:
Oh sure, to sounds better, thanks.